dashboard/app/assets/javascripts/utils.js.erb

```diff
@@ -16,3 +16,37 @@ function onURLBlocked(base_url, callback) {
       .css("position", "absolute")
       .appendTo("body");
 }
+
+var DateHelper = {
```

Comment:
this is not localized :(

maybe we should use https://github.com/rmm5t/jquery-timeago